docs/api/app.md

```diff
@@ -1520,3 +1555,11 @@ or Windows [WOW](https://en.wikipedia.org/wiki/Windows_on_Windows)).
 You can use this property to prompt users to download the arm64 version of
 your application when they are running the x64 version under Rosetta
 incorrectly.
+
+### `app.notificationsComServerCLSID` _Windows_
+
+A `string` which contains fully formed CLSID for notifications COM server.
+This is necessary only for apps with persistent notifications support.
+The current default value for this property is `empty` (not `nil`). It means that by the default no persistent notifications support is exist.
```

Comment:
This property is binded with ElectronBrowserClient::notifications_com_server_clsid_ in case if it is empty it means the we do not want persistent notifications. Think it is obviously that in case of assigning **nullptr** to std::string we will receive UB potentially with app crash. That is why valid variants of usage that property are bellow:
```
app.notificationsComServerCLSID = '{00000001-0002-0003-0405-060708090001}';
app.notificationsComServerCLSID;
//just do not call
```

Beware: variant `app.notificationsComServerCLSID = nil is restricted and not allowed`